python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -52,188 +54,211 @@ def computeClippedImageStats(im):
     return mean, std, var
 
 
-def getImageGrid(im):
-    """! Convenience function for creating a grid of pixel coordinates covering a 2-d array,
-    using `numpy.meshgrid()`
-    @param im input 2-d numpy array
-    @return two numpy arraysm each with same shape as `im`, containing x- and y- coordinate
-    grid.
-    """
-    xim = np.arange(np.int(-np.floor(im.shape[0]/2.)), np.int(np.floor(im.shape[0]/2)))
-    yim = np.arange(np.int(-np.floor(im.shape[1]/2.)), np.int(np.floor(im.shape[1]/2)))
-    x0im, y0im = np.meshgrid(xim, yim)
-    return x0im, y0im
+class ImageDecorrelationConfig(pexConfig.Config):
+    """! Configuration for image decorrelation"""
+    useScipy = pexConfig.Field(
+        dtype = bool,
+        doc = """If True use scipy for image convolution (debugging option);
+                 if False use afwMath""",
+        default = False
+    )
```

Comment:
Minor suggestions here:
- no spaces around "=" when specifying values (run AutoPep8 to see this and clean it up)
- I suggest that doc strings not include hard returns. Consider this, instead (which uses automatic concatenation of adjacent strings):

```
doc="If True use scipy for image convolution (debugging option); "
    "if False use afwMath",

```